ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-ref.yaml

```diff
@@ -102,9 +102,9 @@ ref:
   - name: CLUSTER_TAGS
     default: ""
     documentation: Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated example - 'foo:bar,bar:baz', The default value is "prowci:${CLUSTER_NAME}".
-  - name: ENABLE_SECTOR
-    default: "false"
-    documentation: Enable the sector to specify the provision shard to provision the hosted control planes (HyperShift) cluster.
+  - name: CLUSTER_SECTOR
```

Comment:
With this ENV change, how about the jobs?

Reply:
The sector jobs are still here. The changes will result in we do not need `ENABLE_SECTOR`, only need `CLUSTER_SECTOR` so that the sector workflows should use the same workflows as the regular.